src/components/Form.js

```diff
@@ -1,21 +1,43 @@
-import React from 'react';
+import React, { useRef } from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books/books';
 
-const Form = () => (
-  <form>
-    <label htmlFor="book-title">
-      <input id="book-title" name="bookTitle" type="text" placeholder="Book Title" />
-    </label>
-    <label htmlFor="select-category">
-      <select id="select-category" name="category">
-        <option> Category </option>
-        <option value="Romance">Romance</option>
-        <option value="suspense">Suspense</option>
-        <option value="fiction">Fiction</option>
-        <option value="action">Action</option>
-      </select>
-    </label>
-    <button type="submit">Add Book</button>
-  </form>
-);
+const Form = () => {
+  const dispatch = useDispatch();
+  const bookTitle = useRef(null);
+  const bookAuthor = useRef(null);
+  const bookCategory = useRef(null);
+  const submitBooksToStore = (event) => {
+    event.preventDefault();
+    const newBook = {
+      title: bookTitle.current.value,
+      author: bookAuthor.current.value,
+      category: bookCategory.current.value,
+      id: bookTitle.current.value,
+    };
```

Comment:
- [ ] in the requirements it is mentioned that the ID of the book is unique, right now if 2 books have the same title and the user deletes one of them both are deleted. below I've added the part of the assignment where this is mentioned:
```
// React component
...
// import useDispatch hook
import { useDispatch } from 'react-redux';
// import your Action Creators
import {addBook, removeBook} from './redux/books/books';

const dispatch = useDispatch();

...

const submitBookToStore = () => {
    const newBook = {
        id, // make sure it's unique
        title,
        author
    }

    // dispatch an action and pass it the newBook object (your action's payload)
    dispatch(addBook(newBook));
}

...

<button onClick={submitBookToStore}>Add Book</button>
```

Reply:
## Hi @mikethreels 

Thanks for the feedback, I didn't consider that possibility. I fixed it now!

Reply:
This issue is well resolved! ✔️